src/rust/ensogl/src/display/scene.rs

```diff
@@ -42,6 +42,9 @@ use enso_frp as frp;
 
 pub trait MouseTarget : Debug + 'static {
     fn mouse_down(&self) -> Option<enso_frp::Source> { None }
+    fn mouse_over(&self) -> Option<enso_frp::Source> { None }
+    fn mouse_leave(&self) -> Option<enso_frp::Source> { None }
+    fn mouse_move_global(&self) -> Option<enso_frp::Source> { None }
```

Comment:
What it is for? The operations here should be for mouse target only. Global mouse move should not be included here.

Reply:
Was a half baked attempt at getting mouse events directly to shapes. Has been removed.